heroku_deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine/git
+FROM node:alpine
```

Comment:
If we use NodeJS, maybe is better to set a concrete versions instead get the latest image, to ensure that all works.

Reply:
I thought about doing this but [Heroku's documentation](https://devcenter.heroku.com/articles/heroku-cli#npm) states:

> Heroku uses very current releases of Node.js and does not back-support older versions.

I also thought to set a concrete version for Heroku CLI installed using yarn, but then we should it every time they update, and it can introduce bugs because our fixed version is no longer compatible with their API.